client/config.go

```diff
@@ -3,6 +3,8 @@ package client
 import (
 	"github.com/spf13/pflag"
 	"github.com/spf13/viper"
+	"os/user"
```

Comment:
Yes, but is `~user` really needed? I went hunting for a tilde expanding lib but instead saw lots of posts from people saying not to do it. Which is fine, but ssh's `IdentityFile` supports it and the obvious thing for people to do is make `IdentityFile` match `public_file_prefix`.

I don't know if `IdentityFile` supports `~user` syntax, but the one person in the world who does that can maintain his own cashier version.

Lib added, code nuked.